docs/DesignTokens/Taxonomy.stories.mdx

```diff
@@ -0,0 +1,113 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Design Tokens Taxonomy" />
+
+# Design Tokens Taxonomy — Work in Progress 🛠
```

Comment:
Is it a work in progress? what else do we need to define here?

Also, not sure we should mark it as is, but rather say that everything is a living document